.golangci.yaml

```diff
@@ -9,7 +9,7 @@ service:
 linters:
   disable-all: true
   enable:
-    - whitespace # Tool for detection of leading and trailing whitespace
+    #- whitespace # Tool for detection of leading and trailing whitespace
```

Comment:
normal to comment it?

Reply:
This seems to be from an old commit, the `whitespace` linter has been enabled in [this PR](https://github.com/zivkovicmilos/gno/pull/14), and is currently enabled on the latest branch commit:

https://github.com/zivkovicmilos/gno/blob/02ce9207dbbb2c2fc97e33e42316a6b9dcc79f84/.golangci.yaml#L14